source/Guides/Real-Time-Operating-System-Setup/Real-Time-Linux/real_time_raspberry_pi_images.md

```diff
@@ -0,0 +1,9 @@
+# Real-time Raspberry PI images
+
+## Introduction
+
+For those users who simply want to test ROS 2 real-time applications in a Raspberry Pi, some ready-to-use images are provided.
```

Comment:
I think this we can put in the very beginning. To avoid a situation that people will start building the kernel, and then will figure out that a ready-to-use binaries already exist

Reply:
@razr I tried moving this to the `Real-Time Linux` index document. Unfortunately, I don't how it's possible to add references for markdown documents. I can only see how to do this with `.rst` format (see https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html). If this is a major issue we could convert all the markdown documents to `.rst`. I would do that in a separate document.